PWGHF/DataModel/CandidateReconstructionTables.h

```diff
@@ -533,6 +533,8 @@ DECLARE_SOA_DYNAMIC_COLUMN(PtV0Pos, ptV0Pos, //!
                            [](float px, float py) { return RecoDecay::pt(px, py); });
 DECLARE_SOA_DYNAMIC_COLUMN(PtV0Neg, ptV0Neg, //!
                            [](float px, float py) { return RecoDecay::pt(px, py); });
+DECLARE_SOA_DYNAMIC_COLUMN(V0Ct, v0ct, //!
```

Comment:
```suggestion
DECLARE_SOA_DYNAMIC_COLUMN(CtV0, ctV0, //!
```
This seems more consistent with the names of functions where you use this column.

Reply:
Also please add documentation for the columns in that namespace